app/controllers/admin_controller.rb

```diff
@@ -44,7 +44,7 @@ def export_str(shf_apps)
       out_str << ','
 
       # a company name may have commas, so surround with quotes so spreadsheets recognize it as one string and not multiple comma-separated value
-      out_str << (m_app.company.nil? ?  '' : "\"#{m_app.company.name}\"")
+      out_str << (m_app.companies.empty? ?  '' : "\"#{m_app.companies.last.name}\"")
```

Comment:
OK with me if this is OK with @thesuss .

Reply:
Or, we could list a comma-separated list of company names for the application(?)

I did not go that way because I am not sure if that is important for this exported list of apps.